backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/utils/BehaviouralCallback.kt

```diff
@@ -0,0 +1,6 @@
+package net.skyscanner.backpack.compose.utils
+
+interface BehaviouralCallback {
```

Comment:
Should this be combined into one file with the `BehaviouralModifiers` (which is just one extension function)? That way we could also add full inline docs to the class

Reply:
yeah same file makes sense 👍